src/main/java/rife/continuations/instrument/TypesMethodVisitor.java

```diff
@@ -994,8 +1005,6 @@ private void processInstructions(TypesNode node) {
                             ContinuationDebug.LOGGER.finest(repeat("  ", node.getLevel()) + "  DUP2_X1 " + type1);
 
                         context.push(type1);
-                        context.push(type2);
```

Comment:
Please don't rearrange this, working with byte code and frames is confusing enough as it is, splitting contextual pushes out in multiple parts makes it even harder